feature-libs/order/components/order-details/order-overview/order-overview.component.ts

```diff
@@ -16,15 +29,27 @@ import { filter, map } from 'rxjs/operators';
   templateUrl: './order-overview.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class OrderOverviewComponent {
+export class OrderOverviewComponent implements OnInit {
   order: any;
 
   @Input('order')
   set setOrder(order: any) {
     this.order = order;
   }
+  isUserOrgUnitViewer: boolean;
 
-  constructor(protected translation: TranslationService) {}
+  constructor(
+    protected translation: TranslationService,
+    protected userAccountFacade: UserAccountFacade
+  ) {}
+
+  ngOnInit(): void {
+    this.userAccountFacade.get().subscribe((user: User | undefined) => {
+      this.isUserOrgUnitViewer = !!user?.roles?.includes(
```

Comment:
Is the !! needed here? The `includes` method returns a boolean. Maybe your intent is to use it for the user object?

Reply:
it is needed since `user?` returns `boolean | undefined` and this causes a type check error
I changed the variable type